### ISSUE TITLE
Formatting a concatenated string

### ISSUE THREAD

**Issue by EricTurf** [ericadamski/knex-case#2]
When using a concatenated string is wraps it in single quotes, this should be changed to return the string not wrapped in strings.

ex:

`
then(`I + am + eric`) returns 'I + am + eric' and it should return I + am + eric
`

**Comment by ericadamski** [comment]
A possible solution to this is to escape any formatting with `knex.raw` for example:

```
const expected = `(CASE WHEN column=1 THEN (i + am + eric) ELSE 0 END)`;

const result = K.queryBuilder()
          .when('column', '=', 1)
          .thenElse(K.raw('(i + am + eric)'), 0)
          .toQuery();

expect(result).toBe(expected);
```

What do you think @EricTurf ?

**Comment by EricTurf** [comment]
Ya I think we probably should leave that responsibility to the user. There would probably be too many cases for us to properly format everything properly based on the scenario

**Comment by ericadamski** [comment]
This should have been addressed in 0c62ab467ac26dc6bee2b7628a8bf3bc68465475
